### PR TITLE
Added Status Effects. Refined Astromancer. Closes #137.

### LINKED ISSUE #137: DeckBuilder: Display Decklist for Selected Character
Like Decklist on right: https://techraptor.net/wp-content/uploads/2019/04/steamworld-quest-deck-building.jpg

### PR DESCRIPTION
**Pull request by Kazamaru**

### CODE REVIEW

**Comment by 0crat** [comment]
@silasreinagel[/z](https://www.0crat.com/u/silasreinagel) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#160](https://github.com/enigmadragons/projectneon/issues/160) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@silasreinagel[/z](https://www.0crat.com/u/silasreinagel) everybody who has role `REV` is banned at [#160](https://github.com/enigmadragons/projectneon/issues/160); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

File: design/AstroMancer.md [hunk 1]
```diff
@@ -14,25 +14,32 @@ Spell-Reliant
...
 - Apply status effects
 - Be single target
 - Be high impact
 
 
+##Paragon
+
+The Stellar path makes Lunar cards feed into Stellar cards.(Often granting bonuses to your team.)
+
+The Solar path makes all Solar cards become modal allowing you to choose between 2 options.(The primary, and a secondary that cannot
```
Comment (SilasReinagel) [inline design/AstroMancer.md]: This last sentence is incomplete.

**Comment by 0crat** [comment]
Job `gh:enigmadragons/projectneon#160` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#160](https://github.com/enigmadragons/projectneon/issues/160) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @silasreinagel[/z](https://www.0crat.com/u/silasreinagel)

**Pull request merged**